src/blog/2025/03/deploy-node-red-flows-mqtt-brokers-flowfuse.md

```diff
@@ -0,0 +1,153 @@
+
+However, when multiple remote instances exist within the same stage, such as in a device group, configuring them individually can be impractical, especially when most settings remain the same. To simplify this process, FlowFuse allows you to set environment variables at the device group level, and you can use the second method to make the configuration quick and easy while using remote instance default environment variables.
+
+To add the device group level environment variables, follow the steps:
+
+1. Go to your **device group’s settings** in the FlowFuse platform.
+2. Add the following environment variables with the appropriate values:
+
+   - `HOST`
+   - `PORT`
+   - `CLIENT_ID_SUFFIX`
+
+3. Click **Save Settings** to apply the changes.
+
+![Configuring environment variables at the device group level.](./images/env-settings.png){data-zoomable}
+_Configuring environment variables at the device group level._
+
+With this approach, you do not need to set environment variables separately for each instance. Instead, the environment variables defined at the device group level will apply to all remote instances within the group, ensuring a consistent and efficient configuration.
+
+## Deploying Flow with Stage-Specific Configurations via DevOps Pipeline
+
+Now that everything is set up trigger the deployment pipeline for the development stage. This will push the flow and settings to the next stage while preserving the existing environment variables. Before applying the new configuration, all remote instances in the target stage will restart automatically.
+
+![Deploying Node-RED flows using FlowFuse's DevOps pipeline.](./images/devops-pipeline.png){data-zoomable}
+_Deploying Node-RED flows using FlowFuse's DevOps pipeline._
+
+If everything is configured correctly, the MQTT nodes in each remote Node-RED instance will connect to the appropriate broker configured at the environment settings.
+
+## Monitoring MQTT Topic Hierarchy with FlowFuse
+
+Once the Node-RED flow is deployed on all remote instances in the device group and the MQTT nodes in each instance are connected to the broker, you can monitor and manage all topics across all brokers directly in FlowFuse.
+
+![Monitoring MQTT topics in FlowFuse across multiple brokers.](./images/broker-topic-monitoring.png){data-zoomable}
+_Monitoring MQTT topics in FlowFuse across multiple brokers._
+
+Watch this short video to learn how to bring your brokers for topic monitoring in FlowFuse: [https://youtube.com/shorts/-8TPXb0h0vA?si=wi3wghi4vUWlXJTZ](https://youtube.com/shorts/-8TPXb0h0vA?si=wi3wghi4vUWlXJTZ)
+
+## Conclusion
+
+FlowFuse makes it easy to deploy Node-RED flows while ensuring each stage connects to the right MQTT broker. You can automate MQTT configurations by using environment variables at both the instance and device group levels, reducing manual setup and ensuring consistency.
+
+[Get Started Now](https://app.flowfuse.com/account/create/)
```

Comment:
@sumitshinde-84 Start thinking about a CTA banner in a partial to include in blog posts.